include/native/reduce.h

```diff
@@ -6,7 +6,13 @@
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
-*
+*te < typename InputIt, typename ReduceOperator>
```

Comment:
I imagine this is a mistake. What's the point of this in the middle of the license?